chipy_org/apps/meetings/tests/test_rsvps.py

```diff
@@ -109,7 +114,9 @@ def test_unique_rsvp(self):
         Tests the uniqueness constraints on the rsvp model
         """
         test_venue = Venue.objects.create(name="Test")
-        meeting = Meeting.objects.create(when=datetime.date.today(), where=test_venue)
+        meeting = Meeting.objects.create(
+            when=datetime.date.today(), where=test_venue, in_person_capacity=5
```

Comment:
It seems like we're using `where` too many places. As you work in here feel free to remove anything that doesn't need to assert things about `where`.

Reply:
:+1: